docs/pages/management/infrastructure-as-code/terraform-starter/enroll-resources.mdx

```diff
@@ -0,0 +1,749 @@
+         "env" = "dev"
+       }
+     }
+   
+     spec = {
+       uri = "example.com:3000"
+     }
+   }
+   ```
+
+   1. Update the `spec.uri` field to match your application's domain name and
+      port.
+   
+   1. Read the [Terraform Provider
+      Reference](../../reference/terraform-provider/resources/app.mdx) for the
+      `teleport_app` resource and adjust any additional configuration fields.
+   
+   </TabItem>
+   <TabItem label="Database">
+   
+   ```hcl
+   resource "teleport_database" "example" {
+     metadata = {
+       name        = "example"
+       description = "Test database"
+       labels = {
+         // Teleport adds this label by default, so add it here to
+         // ensure a consistent state.
+         "teleport.dev/origin" = "dynamic"
+         // Change "dev" to "prod" as appropriate.
+         "env" = "dev"
+       }
+     }
+   
+     spec = {
+       protocol = "postgres"
+       uri      = "localhost"
+     }
+   }
+   ```
+
+   1. Update the `protocol` field to match the protocol of your database. Run the
+      following command on your workstation to see all supported protocols:
+
+      ```code
+      $ teleport help db configure create
+      ```
+
+   1. Update the `uri` field to match the domain name port of your database. If
+      you manage your database with Terraform, you can set it to an attribute of
+      the corresponding Terraform resource to establish a dependency
+      relationship.
+   1. Read the [Terraform Provider
+      Reference](../../reference/terraform-provider/resources/database.mdx) for
+      the `teleport_database` resource and adjust any additional configuration
+      fields.
+   
+   </TabItem>
+   <TabItem label="Agentless OpenSSH Server">
+
+   ```hcl
+   resource "teleport_server" "openssh_agentless" {
+     version = "v2"
+     sub_kind = "openssh"
+     metadata = {
+       name        = "example"
+       description = "Test server"
+       labels = {
+           // Teleport adds this label by default, so add it here to
+           // ensure a consistent state.
+           "teleport.dev/origin" = "dynamic"
+           // Change "dev" to "prod" as appropriate.
+           "env" = "dev"
+         }
+       }
+     spec = {
+       addr = "198.51.100.1:22"
+       hostname = "example"
+     }
+   }
+   ```
+
+   1. Update `spec.addr` with the domain name or IP address of the server as well
+      as the port. If you manage your database with Terraform, you can set it to
+      an attribute of the corresponding Terraform resource to establish a
+      dependency relationship.
+   1. Update `hostname` with the hostname of the server as shown by Teleport
+      client tools (Teleport Connect, `tsh`, and the Web UI). 
+   1. Read the [Terraform Provider
+      Reference](../../reference/terraform-provider/resources/server.mdx) for
+      the `teleport_database` resource and adjust any additional configuration
+      fields.
+
+   </TabItem>
```

Comment:
I don't think those instructions are enough to successfully enroll an agentless openssh node? Agentless SSH has many prerequisites including:
- trusting the teleport CA on the openssh node
- have Teleport issue SSH certificates for openssh and load them on the node
- the node must be publicly reachable from the proxy

I think it would be better to link to not mention agentless SSH here as the subject is too complex and the guide does not provide enough information for a user to achieve a successful setup.